include/flatbuffers/reflection.h

```diff
@@ -428,6 +428,11 @@ Offset<const Table *> CopyTable(FlatBufferBuilder &fbb,
                                 const Table &table,
                                 bool use_string_pooling = false);
 
+bool Verify(flatbuffers::Verifier &v,
+            const reflection::Schema &schema,
+            const reflection::Object &obj,
+            const flatbuffers::Table &table);
```

Comment:
it be better to pass a buffer `(const uint8_t *, size_t)` instead of a `Table`, since the latter fails to verify the root pointer. If you do that, then you can also create the Verifier internal to this call.